package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.testserver",
-  "version": "3.3.4",
+  "version": "3.4.0",
```

```suggestion
  "version": "3.3.5",
```
We've been using patch version when just changing/adding the specs. Keep minor for larger changes. Not really following semver here.